packages/flutter/lib/src/material/ink_decoration.dart

```diff
@@ -251,9 +251,21 @@ class _InkState extends State<Ink> {
 
   @override
   void deactivate() {
+    _ink?.visible = false;
```

Comment:
Instead of a property it feels like there should be imperative hide/show methods on the ink.